### DISCUSSION TITLE
Mermaid Zoom Buttons

### DISCUSSION THREAD

**Discussion by SPyofgame** [squidfunk/mkdocs-material#5237] [Q&A] [answered]
Is it possible to have those buttons in mkdocs ?

![image](https://user-images.githubusercontent.com/62207095/226519836-913cb147-8a62-430d-ad50-3ce93a6056d7.png)

**Comment by squidfunk** [comment, marked as answer]
Duplicate of #5208 – please use the search functionality.

**Reply by SPyofgame** [reply]
Sorry, I was too enthusiastic to try to use all features I got from you and facelessuser 😆

**Reply by SPyofgame** [reply]
Does it seem that there is little or no documentation to use all features from glightbox ?

**Reply by squidfunk** [reply]
[mkdocs-glightbox](https://github.com/blueswen/mkdocs-glightbox) is a third-party plugin that is general and not tied to Material for MkDocs. Please consult [their documentation](https://blueswen.github.io/mkdocs-glightbox/) for configuration options. We don't implement everything in the MkDocs ecosystem, there are several projects we build upon 😊

**Reply by SPyofgame** [reply]
Thank you 🤗
